src/StoryTeller.Testing/Model/Persistence/loading_fixture_with_markdown.cs

```diff
@@ -0,0 +1,138 @@
+﻿using Baseline;
+using Shouldly;
+using StoryTeller.Model;
+using StoryTeller.Model.Persistence.DSL;
+using Xunit;
+
+namespace StoryTeller.Testing.Model.Persistence
+{
+    public class loading_fixture_with_markdown
+    {
+        [Fact]
+        public void reads_title()
+        {
+            var result = FixtureReader.ReadFrom(@"# a title");
+            result.title.ShouldBe("a title");
+        }
+
+        [Fact]
+        public void reads_sentence()
+        {
+            var result = FixtureReader.ReadFrom(@"
+## a title
```

Comment:
Can we flip the "key" and "title"? It might really take precedence, and I might say that we make "title" be optional and just derived from the key if it's missing.